lib/chef-dk/command/shell_init.rb

```diff
@@ -106,7 +106,7 @@ def run(argv)
           export(shell_name, var_name, value)
         end
 
-        msg(completion_for(shell_name))
+        msg(completion_for(shell_name)) unless shell_name == "powershell"
```

Comment:
Sure... it just seemed cleaner to not even call completion_for as we are not completing PowerShell commands.  

Are you proposing:
msg(completion_for(shell_name)) unless completion_for(shell_name).empty?